cmd/erasure-common.go

```diff
@@ -40,9 +42,6 @@ func (er erasureObjects) getLoadBalancedLocalDisks() (newDisks []StorageAPI) {
 func (er erasureObjects) getLoadBalancedNDisks(ndisks int) (newDisks []StorageAPI) {
 	disks := er.getLoadBalancedDisks()
 	for _, disk := range disks {
-		if disk == nil {
```

Comment:
This looks useful for me, at least it will return the maximum of online disks

Reply:
it will never return `disk == nil`